src/gc/env/gcenv.ee.h

```diff
@@ -68,6 +68,12 @@ class GCToEEInterface
     static void StompWriteBarrier(WriteBarrierParameters* args);
 
     static void EnableFinalization(bool foundFinalizers);
+
+    // Global alloc context retrieval
+    static gc_alloc_context *GetGlobalAllocContext();
```

Comment:
Do we need two APIs for this? Can this be just one API - that returns null if the EE is not using global alloc context?